documentation/usage/installation.md

```diff
@@ -2,18 +2,153 @@
+  <div>
+
+![Launching the installer](/docs/installation/opengoal_install_start.png)
+![Choose a location to install OpenGOAL](/docs/installation/opengoal_install_location.png)
+![Start the install](/docs/installation/opengoal_install_confirm.png)
+![Wait for the installer to complete](/docs/installation/opengoal_install_in_progress.png)
+![Launch OpenGOAL once the install is complete](/docs/installation/opengoal_install_finished.png)
+
+  </div>
+</details>
+
+If you choose to launch OpenGOAL once the install is completed, you'll skip right to the next step. If not you'll need to navigate to the location you chose for OpenGOAL to install (by default this location is `C:\Program Files\OpenGOAL-Launcher`), and run `OpenGOAL-Launcher.exe`.
+
+If this opens successfully, you've reached the final stage! Up next we'll be setting up the launcher, including extracting the assets from your `.iso` file.
+
+### Setting up the launcher
+
+This step is very straightforward, but as this project is still in active development you may come across issues along the way. If you hit issues please remember that this is a community led project, and we can't guarantee support. Please first visit the [FAQs](/faq.md) and if you can't find an answer there, you might be able to find help in the OpenGOAL [discord](https://discord.gg/VZbXMHXzWv), but we do not guarantee support.
+
+For this step we'll first give OpenGOAL the location of the `.iso` file we created earlier. Click the `SETUP` button in the launcher window to begin with.
+
+![Setup OpenGOAL with your .iso file](/docs/installation/opengoal_launcher_jd1_setup.png)
+
+This will open a window, and you'll need to navigate to the `.iso` file you created earlier in the setup here to continue. Once you've chosen the `.iso` file, the launcher will begin extracting the assets for the game.
+
+![Launcher extracting assets and completing compilation](/docs/installation/opengoal_launcher_jd1_compiling.png)
+
+This can take some time, but if you find it taking more than a few minutes try opening the `INSTALLATION LOGS` view below the progress bar, and see if there is still progress being made. If you don't see any changes here after a minute or two more, try closing and re-running the launcher.
+
+This process will complete loading the assets from your `.iso` file, and once it's done you'll be ready to run the game!
+
+![Starting Jak and Daxter in OpenGOAL](/docs/installation/opengoal_launcher_jd1_ready_to_start.png)
+
+Once you've reached this screen, you're all set up and ready to go! Just hit `PLAY` to start the game. The other controls are not something you'll need to play the game, as most of them are tools for developers, but do take note of the `SETTINGS AND SAVES` button if you want a link to your saves to move them or back them up, and of course the `UNINSTALL` button if you choose to do so down the line. The OpenGOAL launcher can also be uninstalled from the OpenGOAL installer used earlier in the instructions.
```

Comment:
it'd be nice to include a screenshot of an example of requirements not being met, as this seems to confuse some users even though it's fairly obvious (you dont meet the requirements).  Here's a recent example that could be used:
![image](https://user-images.githubusercontent.com/13153231/198849707-8a2c3847-7745-4872-b8b0-ffda5a091d48.png)

Reply:
Cheers I've now included this image and warning at the top of the setup instructions, hopefully it'll help catch people early! Thanks for this :D